### DISCUSSION TITLE
How to drag a node while keep the other nodes fixed ？

### DISCUSSION THREAD

**Discussion by liumengyu0817** [neo4j-contrib/neovis.js#261] [Q&A]
When I use the example code to get the image, the nodes in the generated image is very dense. Thus, I tried to drag the nodes so that they can become sparse. But, I found that when I dray one node, the other nodes also move. Is there any way to avoid this problem ?

**Comment by thebestnom** [comment]
I don't think it avialible at master, but in 2.0.0 you can do
```js
const config = {
  //normal neovis config
  visConfig: {
     physics: {
       enabled: false
    }
  }
};
```
and it should be enogh if Im not wrong

**Comment by pavelb97** [comment]
> I don't think it avialible at master, but in 2.0.0 you can do
> 
> ```js
> const config = {
>   //normal neovis config
>   visConfig: {
>      physics: {
>        enabled: false
>     }
>   }
> };
> ```
> 
> and it should be enogh if Im not wrong


I've tried this in 2.0 and this doesn't seem to work - any suggestions?

**Comment by ooker777** [comment]
@pavelb97 it should be enough. Can you post your whole code and what's going on in the console?
